src/Model/PaymentCard.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace ThePay\ApiClient\Model;
+
+use DateTime;
+use ThePay\ApiClient\Utils\Json;
+
+class PaymentCard
+{
+    public const TYPE_CREDIT = 'credit';
+    public const TYPE_DEBIT = 'debit';
+    public const TYPE_PREPAID = 'prepaid';
+    public const TYPE_UNKNOWN = 'unknown';
+
+    private string $number;
+    private ?DateTime $expiration_date = null;
+    private ?string $brand;
+    private string $type;
+
+    /**
+     * @param string|array{number:string,expiration_date:DateTime,brand:string,type:self::TYPE_*} $values Json in string or associative array
```

Comment:
I just tried to check expiration date and I encountered en error in annotation: expiration_date has to be string|null instead of DateTime.

I created a test for expiration_date with fix this issue in here:
https://github.com/ThePay/api-client/pull/60